Automatic Validation ended with:
> 2024-05-06T21:56:25.6625051Z ##[error]Manifest Validation Failed
 2024-05-06T21:56:30.7080903Z ##[section]Finishing: Validate Manifest

(Automated response - build 880.)

--- a/manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml
+++ b/manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml
@@ -7,7 +7,7 @@ InstallerLocale: en-US
 InstallerType: wix
 ProductCode: '{8905B882-D647-4593-946E-5537499CCFBA}'
 AppsAndFeaturesEntries:
-  DisplayVersion 1.0.0
+- DisplayVersion: 1.0.0
 Installers:
 - InstallerUrl: https://github.com/Vleerapp/Vleer/releases/download/v1.0.0-beta.3/Vleer_1.0.0-beta.3_x64-setup.msi
   Architecture: x64

